src/Microsoft.Health.Fhir.Core/Features/Operations/Export/CreateExportRequestHandler.cs

```diff
@@ -77,6 +77,29 @@ public async Task<CreateExportResponse> Handle(CreateExportRequest request, Canc
             // Otherwise, we will create a new export job. This will be a best effort since the likelihood of this happen should be small.
             ExportJobOutcome outcome = await _fhirOperationDataStore.GetExportJobByHashAsync(hash, cancellationToken);
 
+            var filters = new List<ExportJobFilter>();
+
+            if (!string.IsNullOrWhiteSpace(request.Filters))
+            {
+                var filterArray = request.Filters.Split(",");
+                foreach (string filter in filterArray)
+                {
+                    var parameterIndex = filter.IndexOf("?", StringComparison.Ordinal);
+                    var filterType = filter.Substring(0, parameterIndex);
```

Comment:
This can be -1, which will throw ArgumentOutOfRangeException.
This whole method need to be wrapped in try/catch and handled properly in case of bad input.

Reply:
I'm adding in a catch for a badly formatted parameter. So long as the format is correct (x?y=z) I think it should allow the input, even if it will fail the search latter on because of an invalid resource type or search parameters.